File: test/ayush.dev/python/pipelines/python_external_git_pipeline/code/python_external_git_pipeline/graph/src_json.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from python_external_git_pipeline.config.ConfigStore import *
from python_external_git_pipeline.udfs.UDFs import *

def src_json(spark: SparkSession) -> DataFrame:
    return spark.read\
        .format("json")\
        .schema(
          StructType([
            StructField("account_flags", StringType(), True), StructField("account_open_date", StringType(), True), StructField("country_code", StringType(), True), StructField("customer_id", StringType(), True), StructField("email", StringType(), True), StructField("first_name", StringType(), True), StructField("last_name", StringType(), True), StructField("phone", StringType(), True)
        ])
        )\
        .load("dbfs:/Prophecy/qa_data/json/CustomersDatasetInput.json")

File: test/ayush.dev/scala/pipelines/scala_external_pipeline/code/src/main/scala/io/prophecy/pipelines/scala_external_git_pipeline/graph/src_json.scala
package io.prophecy.pipelines.scala_external_git_pipeline.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_external_git_pipeline.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object src_json {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("json")
      .schema(
        StructType(
          Array(
            StructField("account_flags",     StringType, true),
            StructField("account_open_date", StringType, true),
            StructField("country_code",      StringType, true),
            StructField("customer_id",       StringType, true),
            StructField("email",             StringType, true),
            StructField("first_name",        StringType, true),
            StructField("last_name",         StringType, true),
            StructField("phone",             StringType, true)
          )
        )
      )
      .load("dbfs:/Prophecy/qa_data/json/CustomersDatasetInput.json")

}
